### PR TITLE
chore: breadcrumb component refactor 

### PR DESCRIPTION
**Pull request by veds-g**
Refactoring the BreadCrumb component, laid out the `common` directory for the same
Changed separator from `/` to `>`. Also highlighted the current view in Breadcrumb

Before
image

After
image

### CODE REVIEW

**Comment by veds-g** [comment]
@whynowy While refactoring, I didn't consider the `case 7` from removed `BreadCrumbs.tsx`- line 28. I believe we don't have `pods-view` pages just `namespace` and `pipeline-view` are needed

File: ui/src/components/common/Breadcrumbs/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <BrowserRouter>
+        <Breadcrumbs />
+      </BrowserRouter>
+    );
+    expect(screen.getByTestId("namespace-breadcrumb")).toBeInTheDocument();
+    expect(screen.getByTestId("mui-breadcrumbs")).toBeInTheDocument();
+  });
+});
+
+// jest.mock("react-router-dom", () => ({
```
Comment (whynowy) [inline ui/src/components/common/Breadcrumbs/index.test.tsx:24]: What is the reason to keep these commented lines?

Comment (veds-g) [reply ui/src/components/common/Breadcrumbs/index.test.tsx:24]: have to fix an issue in the test cases(commented code), Shakira requested to create a branch for simultaneous testing.

File: ui/src/components/common/Breadcrumbs/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+let mockPATH = "";
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useLocation: () => ({
+    pathname: mockPATH,
+  }),
+}));
+
+describe("Breadcrumbs", () => {
```
Comment (mshakira) [inline ui/src/components/common/Breadcrumbs/index.test.tsx:13]: we can have one describe section with multiple it blcoks

File: ui/src/components/common/Breadcrumbs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          className="Breadcrumbs-typ"
+          sx={{ fontWeight: 600 }}
+        >
+          Namespaces
+        </Typography>
+      );
+    }
+    const pathParts = pathname.split("/");
+    switch (pathParts.length) {
+      case 5: // pipeline view
```
Comment (mshakira) [inline ui/src/components/common/Breadcrumbs/index.tsx:27]: can we test it with different paths.

File: ui/src/components/common/Breadcrumbs/index.test.tsx (hunk 2 above)
Comment (veds-g) [reply ui/src/components/common/Breadcrumbs/index.test.tsx:13]: will move `it` under a single block

File: ui/src/components/common/Breadcrumbs/index.tsx (hunk 3 above)
Comment (veds-g) [reply ui/src/components/common/Breadcrumbs/index.tsx:27]: i covered different paths in different `it`'s

**Review by mshakira — APPROVED** [review]

**Review by whynowy — APPROVED** [review]

**Pull request merged**